packages/react-core/src/components/Form/Form.tsx

```diff
@@ -38,8 +41,14 @@ export const Form: React.FunctionComponent<FormProps> = ({
       (isWidthLimited || maxWidth) && styles.modifiers.limitWidth,
       className
     )}
+    ref={innerRef}
   >
     {children}
   </form>
 );
+
+export const Form = React.forwardRef<HTMLFormElement, FormProps>((props, ref) => (
```

Comment:
I think the types are not quite declared correctly here. What if you try:
`export const Form = React.forwardRef((props: FormProps, ref: React.Ref<any>) => (`